apps/web/src/components/Settings/Dispatcher/ToggleDispatcher.tsx

```diff
@@ -22,14 +24,54 @@ interface Props {
 }
 
 const ToggleDispatcher: FC<Props> = ({ buttonSize = 'md' }) => {
+  const { data: idKitData } = useIsIdKitVerifiedQuery();
   const userSigNonce = useAppStore((state) => state.userSigNonce);
   const setUserSigNonce = useAppStore((state) => state.setUserSigNonce);
   const currentProfile = useAppStore((state) => state.currentProfile);
   const canUseRelay = currentProfile?.dispatcher?.canUseRelay;
 
+  const { setOpen: setIDKitOpen } = useIDKit({
+    autoClose: true,
+    enableTelemetry: true,
+    actionId: IDKIT_ACTION_ID,
+    handleVerify: async (result: ISuccessResult) => {
+      const response = await fetch(IDKIT_BRIDGE, {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify({
+          ...result,
+          is_production: IS_MAINNET,
+          action_id: IDKIT_ACTION_ID,
+          signal: currentProfile?.ownedBy
+        })
+      });
+
+      if (response.ok) {
+        return;
+      }
+
+      if (response.status === 400 && (await response.json()).code === 'already_verified') {
+        throw new Error(
+          'You have already verified this phone number with Lens. You can only verify one wallet with one phone number.'
```

Comment:
If we localize this will it work? as you mentioned this will be displayed inside the widget 🙇🏼

Reply:
I'm not super familiar with how the translation system Lenster uses works, but if it'll just return a string in the proper language it should.

Reply:
Ahh ok, then leave as it is, I'll experiment it later 🙇🏼